.env.test

```diff
@@ -6,4 +6,9 @@ FLASK_SKIP_DOTENV=True
 SECRET_KEY=secret-key
 TRELLO_KEY=key
 TRELLO_TOKEN=TRELLO_TOKEN
-TRELLO_BOARD=board_id
+TRELLO_BOARD=test_board2
+
+MONGO_USERNAME=jorbefer
+MONGO_PASS=R07IFduVgPYjyBHn
```

Comment:
This looks like a real set of credentials - we shouldn't be checking real credentials in! Your end-to-end test should still be loading these from `.env` locally, or being passed the correct environment variables in Travis
Your integration test should mock out the database, for example using the `MongoMock` library, your integration test might change the client fixture to look something like:
```python
@pytest.fixture
def client():
 # Use our test integration config instead of the 'real' version
    file_path = find_dotenv('.env.test')
    load_dotenv(file_path, override=True)
 # Create the new app.

 # Use the app to create a test_client that can be used in our
    with mongomock.patch(servers=(('fakemongo.com', 27017),)):
        test_app = todo_app.app.create_app()
        with test_app.test_client() as client:
            yield client
```

And you could then change these checked in values to be fake login/password, and the `MONGO_HOST` would be `fakemongo.com`
Note MongoMock expects the protocol to be `mongodb://` rather than `mongodb+srv` so you would also need to extract out the protocol into an environment variable so it could vary. This might actually be a nice opportunity to roll most of these together into a `connection string` environment variable - that way if there are any other parts that need to vary in future (e.g. query parameters) then it's easy to do so without a code change

Reply:
Hi I have used mongomock, but in a slightly different way for my integration test. Now my integratino test works fine. thanks